app/rdf/query-search.ts

```diff
@@ -342,4 +436,5 @@ const mkScoresQuery = (
       }
   `.GROUP().BY`?iri`.THEN.BY`?title`.THEN.BY`?status`.THEN.BY`?datePublished`
     .THEN.BY`?description`.THEN.BY`?publisher`.THEN.BY`?creatorIri`.THEN
-    .BY`?creatorLabel`.prologue`${pragmas}`.prologue`#pragma join.bind off`; // HOTFIX WRT Stardog v9.2.1 bug see https://control.vshn.net/tickets/sbar-1066
+    .BY`?creatorLabel`.prologue`${pragmas}`.prologue`#pragma join.bind off`;
```

Comment:
There's not a need to group by `termsetIri` and `termsetLabel` when `searchingSharedDimensions`? 🤔

Reply:
No, actually since we concat them in the select clause, we should not group by.